2019-09-20 11:15:54,103 server.py > server:DEBUG Получено сообщение от клиента ('127.0.0.1', 60618): {'action': 'presence', 'time': 1568967354.103229, 'type': 'status', 'user': {'account_name': 'Guest', 'status': None}}
2019-09-20 11:16:25,717 server.py > server:DEBUG Получено сообщение от клиента ('127.0.0.1', 60630): {'action': 'presence', 'time': 1568967385.716998, 'type': 'status', 'user': {'account_name': 'Guest', 'status': None}}
2019-09-20 11:16:54,955 server.py > server:DEBUG Получено сообщение от клиента ('127.0.0.1', 60635): {'action': 'presences', 'time': 1568967414.955622, 'type': 'status', 'user': {'account_name': 'Guest', 'status': None}}
2019-09-20 11:16:54,956 server.py > server:ERROR Ошибка валидации сообщения от ('127.0.0.1', 60635),  ошибка в сообщении, module 'jim.messages' has no attribute 'presences'
2019-09-20 11:17:00,201 server.py > server:DEBUG Получено сообщение от клиента ('127.0.0.1', 60636): {'action': 'presence', 'time': 1568967420.201659, 'type': 'status', 'user': {'account_name': 'Guest', 'status': None}}
2019-09-20 13:20:53,480 server.py > server:DEBUG Получено сообщение от клиента ('127.0.0.1', 62761): {'action': 'presence', 'time': 1568974853.480466, 'type': 'status', 'user': {'account_name': 'Guest', 'status': None}}
